go.mod

```diff
@@ -3,19 +3,45 @@ module github.com/makerdao/vulcanizedb
 go 1.12
 
 require (
-	github.com/dave/jennifer v1.3.0
 	github.com/ethereum/go-ethereum v1.9.8
```

Comment:
I meant to revisit this - there are WAY more changes to the go.mod here than I think are strictly necessary, and I believe they were added by mistake.